web/blog/2022-06-01-gitpod-hackathon-guide.md

```diff
@@ -0,0 +1,76 @@
+---
+title: How to win a hackathon. Brief manual.
+authors: [maksym36ua]
+image: /img/martijn-intro-banner.png
+tags: [wasp, gitpod, hackathon]
+---
+
```

Comment:
I think for most posts @matijaSos recommends we use:

```
import InBlogCta from './components/InBlogCta';
import WaspIntro from './_wasp-intro.md';

<WaspIntro />
<InBlogCta />
```

May want to here as well for consistency.